app/cdap/components/PipelineTriggers/PipelineListCompositeTab/index.tsx

```diff
@@ -0,0 +1,399 @@
+                pipelineRow={pipeline.name}
+                isExpanded={expandedPipeline === pipeline.name}
+                onToggle={toggleExpandPipeline}
+                triggeringPipelineInfo={triggeringPipelineInfo}
+                triggeredPipelineInfo={triggeredPipelineInfo}
+                selectedNamespace={selectedNamespace}
+                configureError={configureError}
+                pipelineName={pipelineName}
+                triggersGroupToAdd={triggersGroupToAdd}
+                triggersGroupRunArgsToAdd={triggersGroupRunArgsToAdd}
+              />
+            );
+          })}
+        </PipelineListContainer>
+      )}
+      <SelectedGroupPipelinesContainer>
+        <ButtonsWrap>
+          <PipelineTriggerComputeProfileButton
+            onClick={handlePayloadToggleClick}
+            data-cy={`${state.triggerName}-view-payload-btn`}
+            data-testid={`${state.triggerName}-view-payload-btn`}
+          >
+            {T.translate(`${PREFIX}.configComputeProfie`)}
+          </PipelineTriggerComputeProfileButton>
+          <EnableGroupTriggerButton
+            color="primary"
+            disabled={state.isNameInvalid || triggersGroupToAdd.length === 0}
+            variant="contained"
+            onClick={() => addGroupTriggerClick()}
+            data-cy="enable-group-trigger-btn"
+            data-testid="enable-group-trigger-btn"
+          >
+            {T.translate(`${PREFIX}.addNewTrigger`)}
+          </EnableGroupTriggerButton>
+        </ButtonsWrap>
+        <PayloadConfigModal
+          triggeringPipelineInfo={{
+            id: '',
+            namespace: selectedNamespace,
+          }}
+          isOpen={state.computeModalOpen}
+          triggeredPipelineInfo={triggeredPipelineInfo}
+          onConfigureSchedule={configureComputeProfile}
+          configureError={configureError}
+          onToggle={handlePayloadToggleClick}
+          pipelineCompositeTriggersEnabled={true}
+          modalConfigTab={ConfigTabs.ComputeProfileTabConfig}
+        />
+      </SelectedGroupPipelinesContainer>
+    </PipelineListTabDiv>
+  );
+};
+
+const mapStateToProps = (state) => {
+  return {
+    existingTriggers: state.triggers.enabledTriggers,
+    pipelineList: state.triggers.pipelineList,
+    triggersGroupToAdd: state.triggers.triggersGroupToAdd,
+    triggersGroupRunArgsToAdd: state.triggers.triggersGroupRunArgsToAdd,
+    selectedNamespace: state.triggers.selectedNamespace,
+    selectedTriggersType: state.triggers.selectedTriggersType,
+    pipelineName: state.triggers.pipelineName,
+    expandedPipeline: state.triggers.expandedPipeline,
+    configureError: state.triggers.configureError,
+  };
+};
+
+const mapDispatch = (dispatch) => {
+  return {
+    toggleExpandPipeline: (pipeline) => {
+      dispatch({
+        type: PipelineTriggersActions.setExpandedPipeline,
+        payload: { expandedPipeline: pipeline },
+      });
+    },
+  };
+};
+
+const PipelineListCompositeTab = connect(
```

Comment:
Do you still want to use `connect` for new components? or we should switch to using `useSelector` etc

Reply:
I'd stay with the `connect` for now, since the whole sidebar is using the redux store